### DISCUSSION TITLE
In GitHub PRs, "Quote reply" is not keyboard accessible.

### DISCUSSION THREAD

**Discussion by daniel-montalvo** [community/community#118522] [Accessibility]
### Select Topic Area

Bug

### Body

To replicate.

- Open a GitHub issue or PR with a comment.
- Select the "More options" button, and then arrow down until you get to "Quote reply".
- Press Enter.

Expected behavior: The popup closes and the cursor is on the edit field with the contents of the previous comment preceeded by greater signs ">".

Obtained behavior: The popup closes and focus goes back to the "More options" button. Either the mouse or screen reader mouse-click simulation techniques are required to activate "Quote reply".

**Comment by patrickhlauke** [comment]
Confirming that I can see the same bug (strangely, the "Quote reply" seems to work fine here in discussions, but not on issues)

**Comment by queenofcorgis** [comment]
Thank you for this report. It has been logged internally. 

Any additional members who are experiencing the same situation, please comment to share or upvote it.

We appreciate you contributing and continuing to support other users in the GitHub Community Discussions.

**Reply by daniel-montalvo** [reply]
Hi @queenofcorgis 

Thanks for answering back. Do we have an idea as to when this could be addressed?

**Reply by cehfisher** [reply]
Hello @daniel-montalvo - thank you for taking the time to provide this feedback! Our Accessibility team is actively working on this issue. We will follow-up here with any additional questions or updates when we have them. Thanks again!

**Reply by cehfisher** [reply]
Hi @daniel-montalvo - there were some recent updates to the component you had issues with previously. Could you re-test things out on your end to see if they have been resolved for you? Thanks so much!

**Reply by patrickhlauke** [reply]
@cehfisher sadly, from my testing just now, "Quote reply" still doesn't work when you try to activate it with a keyboard (all other commands do work, this is the odd one out)

EDIT: actually, turns out that it works in *issues* (with the new changes to issues), but not *pull requests* (which still use the old structure/component)

**Reply by cehfisher** [reply]
Thanks for the review @patrickhlauke! We have updated the internal tracking issue with this information. We will follow-up with any updates soon.

**Comment by daniel-montalvo** [comment]
Thanks @patrickhlauke 

Yes, it is not working on my side either.

there are other nice improvements to issues that are not making their way into the PR views, like each issue being an h3. It would be great it the issues and PR views could be harmonized.

**Comment by cehfisher** [comment]
Great news @daniel-montalvo and @patrickhlauke - a new experience has been rolled out to replace the inaccessible one, with support from the accessibility team. Could you re-test and share your feedback? Thanks!

**Comment by daniel-montalvo** [comment]
Fixed for me. Thanks all who worked on this!

**Reply by patrickhlauke** [reply]
@cehfisher confirming that it works now - though a side note, it seems the three-dot context menu for pull requests still diverges in its behaviour from the one in issues - in the latter, once you activate the "..." button, the menu is opened and the first item is focused; in pull requests, the first time a user opens the menu, the first item is *not* focused - it appears it takes a fraction of a second for the menu to actually be populated with options, so focus is lost - and user has to tab forward to get back into it. on subsequent activations though, it works as it should (as the menu's options are now populated?)

Video: opening the "..." menu the first time around with the keyboard, user has to `Tab` to reach it. after closing and reopening, the first item in the menu is focused automatically.

https://github.com/user-attachments/assets/bad6f657-8eb7-4b13-82f1-2f67afd8940c

**Reply by cehfisher** [reply]
Thanks for letting us know that the issue is resolved for you @daniel-montalvo! We will be sure to pass on your thanks to the team members that worked on this.

@patrickhlauke - we will add your video and notes to the internal issue for further evaluation. Thanks for taking the time to share this additional info!
